tests/Unit/ProtoTests/BasicDiregapic/out/samples/LibraryClient/add_comments.php

```diff
@@ -40,13 +39,10 @@ function add_comments_sample(string $formattedName): void
 
     // Prepare the request message.
     $comments = [new Comment()];
-    $request = (new AddCommentsRequest())
-        ->setName($formattedName)
-        ->setComments($comments);
```

Comment:
This is awesome!!  This doesn't really belong in this PR, but we should consider adding the resource name helpers to the samples. E.g, this would be
```php
$request = AddCommentsRequest::build($formattedName, $comments);
```

Reply:
> but we should consider adding the resource name helpers

You mean the request builder methods? Yeah you are right. Separate PR tho fosho, and obviously those are in use when the `V2` client is used for sample gen.